docs/synapse/userguides/storm_ref_automation.rstorm

```diff
@@ -154,185 +180,407 @@ Triggers and Cron
+  :ref:`stormlibs-lib-cron-move` library) or deleted if no longer needed.
+ 
+- **Permissions.** Cron jobs execute with the privileges of a designated user (by default, the user who creates the
+  cron job). We strongly encourage the use of least privilege; the cron job's account should have the permissions
+  required to execute the associated Storm, but no more. One option is for organizations to create a dedicated
+  account for use with automation in Synapse.
+  
+  The owner (creator) of a cron job can be modified using the Storm :ref:`stormlibs-lib-cron-get` library and
+  the ``set()`` method of the :ref:`stormprims-cronjob-f527` primitive. For example:
+  
+  ::
     
-    This view-specific behavior is transparent when using a simple Synapse implementation consisting of a
-    single Cortex with a single layer and a single view (Synapse's default configuration).
+    $mycron=$lib.cron.get(<cron_iden>) $mycron.set(creator, <new_creator_iden>)
+  
+  Users (and roles) cannot create or manage cron jobs by default. The various ``cron.*`` permissions must be granted
+  to users or roles that should be allowed to work with cron jobs in your environment. See the `Cortex permissions`_
+  section of the `Synapse Admin Guide`_ for details on the ``cron.*`` permissions and associated gates.
+  
+  .. NOTE::
     
-    In environments with multiple views, and in particular where users may frequently :ref:`gloss-fork`
-    a view) you should take this view-specific behavior into account. Key considerations include
-    determining where (in which view) triggers and cron jobs should reside, and understanding what
-    happens when you merge or delete a view that contains triggers or cron jobs (discussed in more detail
-    in the appropriate sections below).
+    Where a user has **admin** privileges, all permissions checks are bypassed. This means that users can create
+    and manage cron jobs in views that they fork.
+
+- **Managing Cron Jobs.** Cron jobs can be created, viewed, and managed using the various Storm :ref:`storm-cron`
+  commands, the :ref:`stormprims-cronjob-f527` primitive, or the :ref:`stormlibs-lib-cron` libraries. In :ref:`gloss-optic`,
+  cron jobs can also be created and managed using the :ref:`gloss-admin-tool`.
+  
+  When a cron job is created, Synapse generates a ``syn:cron`` runtime node ("runt node") with some details about
+  the cron job (including the associated Storm) available as properties on the node. While runt nodes are typically
+  read-only, ``syn:cron`` nodes include ``:name`` and ``:doc`` secondary properties that can be set and modified via
+  Storm. This allows you to manage cron jobs by giving them meaningful names and descriptions. Changes to these
+  properties will persist after a Cortex restart.
+  
+  Runt nodes can be lifted, viewed, filtered, etc. just like other nodes in Synapse, which allows you to perform
+  limited introspection of cron jobs using Storm.
```

Comment:
Currently we are planning on removing `syn:cron` and `syn:trigger` runt nodes in 3.x

Reply:
Lets remove the ref to run nodes for cron and trigger if possible

Reply:
Removed